src/cockpit/bridge.py

```diff
@@ -208,6 +208,24 @@ def main() -> None:
     parser.add_argument('--version', action='store_true', help='Show Cockpit version information')
     args = parser.parse_args()
 
+    polyfill = sys.version_info < (3, 7, 0) and not hasattr(asyncio, 'run')
+    if polyfill:
+        # Polyfills for Python 3.6:
+        loop = asyncio.get_event_loop()
+
+        assert not hasattr(asyncio, 'get_running_loop')
+        asyncio.get_running_loop = lambda: loop
+
+        assert not hasattr(asyncio, 'create_task')
+        asyncio.create_task = loop.create_task
+
+        assert not hasattr(asyncio, 'run')
+
+        def asyncio_run(args, debug=False):
+            loop.run_until_complete(args)
+
+        asyncio.run = asyncio_run
+
```

Comment:
This shouldn't be necessary.  We should just call `systemd_ctypes.asyncio_run()`.  That's what it was intended for.

Reply:
Note: these shims have a bug in them.  But it's a systemd_ctypes issue.  Specifically, the `run()` replacement doesn't cancel (and wait on) outstanding asynchronous generators — it only waits on the main task.

Reply:
Done. Works fine, but this is now blocked on the systemd_ctypes PR, marked as such.